spec/models/customer_spec.rb

```diff
@@ -1,5 +1,140 @@
+    @transaction_23 = @invoice_8.transactions.create!(credit_card_number: '4540842003561938', credit_card_expiration_date: '09/22/20', result: 'failed')
+    @transaction_24 = @invoice_8.transactions.create!(credit_card_number: '4540842003561938', credit_card_expiration_date: '09/22/20', result: 'failed')
+
+    @transaction_25 = @invoice_9.transactions.create!(credit_card_number: '4140149827486249', credit_card_expiration_date: '10/22/20', result: 'failed')
+    @transaction_26 = @invoice_9.transactions.create!(credit_card_number: '4140149827486249', credit_card_expiration_date: '10/22/20', result: 'failed')
+    @transaction_27 = @invoice_9.transactions.create!(credit_card_number: '4140149827486249', credit_card_expiration_date: '10/22/20', result: 'failed')
+
+    @transaction_28 = @invoice_10.transactions.create!(credit_card_number: '4923661117104166', credit_card_expiration_date: '08/22/20', result: 'success')
+    @transaction_29 = @invoice_10.transactions.create!(credit_card_number: '4923661117104166', credit_card_expiration_date: '08/22/20', result: 'success')
+    @transaction_30 = @invoice_10.transactions.create!(credit_card_number: '4923661117104166', credit_card_expiration_date: '08/22/20', result: 'success')
+
+    @transaction_31 = @invoice_11.transactions.create!(credit_card_number: '4923661117104166', credit_card_expiration_date: '08/22/20', result: 'success')
+    @transaction_32 = @invoice_11.transactions.create!(credit_card_number: '4923661117104166', credit_card_expiration_date: '08/22/20', result: 'success')
+    @transaction_33 = @invoice_11.transactions.create!(credit_card_number: '4923661117104166', credit_card_expiration_date: '08/22/20', result: 'success')
+
+    @transaction_34 = @invoice_12.transactions.create!(credit_card_number: '4923661117104166', credit_card_expiration_date: '08/22/20', result: 'success')
+    @transaction_35 = @invoice_12.transactions.create!(credit_card_number: '4923661117104166', credit_card_expiration_date: '08/22/20', result: 'success')
+    @transaction_36 = @invoice_12.transactions.create!(credit_card_number: '4923661117104166', credit_card_expiration_date: '08/22/20', result: 'success')
+  end
+
+  describe "relationships" do 
+    it {should have_many :invoices}
+    it {should have_many(:transactions).through(:invoices)}
+    it {should have_many(:invoice_items).through(:invoices)}
+    it {should have_many(:items).through(:invoice_items)}
+    it {should have_many(:merchants).through(:items)}
+  end
+
+  describe "Admin user story set up" do
+
+      #US_21
+
+    describe "class methods" do 
+      describe ".top_customers" do 
+        it 'displays the names of the top 5 customers by successful transactions' do 
+          expect(Customer.top_5_customers).to match([@customer_6, @customer_2, @customer_3, @customer_5, @customer_1])
+          expect(Customer.top_5_customers[0].first_name).to match("Heber")
```

Comment:
Zanna and I wrote a class method for Customer called `.full_name` that stringifies a customer's first+last name, if you wanted to include that here (or after we merge our PR)